containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -24,6 +24,24 @@
 
 #include <../../core/unit_test/tools/include/ToolTestingUtilities.hpp>
 
+//! Get fence event message for @c ZeroMemset.
+template <typename execution_space>
+std::string fence_event_message_for_zeromemset(const execution_space& space) {
+  return std::string(space.name()).append(": ZeroMemset for");
+}
+
+//! @name Some tests are skipped for @c CudaUVM memory space.
+///@{
+#ifdef KOKKOS_ENABLE_CUDA
+#define GTEST_SKIP_IF_CUDAUVM_MEMORY_SPACE                            \
```

Comment:
Other discussion aside, standard practice when defining a macro like this is to wrap the contents in `do { ... } while (false)` so that the use sites have to treat it like a statement with a trailing `;`

Reply:
It is now
```c++
#ifdef KOKKOS_ENABLE_CUDA
#define GTEST_SKIP_IF_CUDAUVM_MEMORY_SPACE                            \
  if constexpr (std::is_same_v<typename TEST_EXECSPACE::memory_space, \
                               Kokkos::CudaUVMSpace>)                 \
    GTEST_SKIP() << "skipping since CudaUVMSpace requires additional fences"
#else
#define GTEST_SKIP_IF_CUDAUVM_MEMORY_SPACE do {} while(false)
#endif

Reply:
I don't agree with Phil's suggestion.  This only apply to macro that take arguments.
I would prefer to revert but I am not blocking on this.

Reply:
I also preferred how it was before. I'll revert.

Reply:
@dalg24 I think this is the last thread. If you agree with all the changes, this PR is good to go :rocket: